understanding/21/reflow.html

```diff
@@ -12,68 +12,197 @@ <h1>Understanding Reflow</h1>
     <section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
-				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
-        <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>
+				<dt>Goal</dt><dd>Content can be enlarged without needing to scroll in two directions.</dd>
+				<dt>What to do</dt><dd>Allow long lines and other related content to reflow within .</dd>
```

Comment:
The end of this looks incomplete; did you mean to drop "the viewport"?

Reply:
sort of?  

in the google doc i mentioned i'm not sure if viewport is really the best word here.   i didn't mean to leave it off in this PR, but also, see prior statement.